packages/fast-components-react-msft/src/flipper/flipper.tsx

```diff
@@ -14,10 +14,15 @@ import {
     ManagedClasses,
 } from "@microsoft/fast-components-class-name-contracts-msft";
 import { DisplayNamePrefix } from "../utilities";
+import { FlipperProps } from ".";
```

Comment:
FYI - I've seen this create pathing issues. I would path this directly to the props file.